nodeup/pkg/model/miscutils.go

```diff
@@ -57,7 +57,7 @@ func (b *MiscUtilsBuilder) Build(c *fi.ModelBuilderContext) error {
 		packages = append(packages, "apt-transport-https")
 	} else if b.Distribution.IsRHELFamily() {
 		packages = append(packages, "curl")
-		packages = append(packages, "python")
+		packages = append(packages, "python2")
```

Comment:
Is this not `python` for Centos7?  Or is python an alias for python2?

Reply:
On CenOS 7 python2 is an alias for python.